### PR TITLE
Use permalink for join_standard_tags integration

### PR DESCRIPTION
**Pull request by a7i**
### What does this PR do?

URL is pointing to the wrong key (should be `join_standard_tags`).
Screen Shot 2022-08-30 at 9 37 51 AM

### Motivation

Incorrect documentation

### Additional Notes

---

### Reviewer checklist
- [x] Review the changed files.
- [x] Review the URLs listed in the [Preview](#preview) section.
- [x] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/getting_started/tagging/unified_service_tagging.md [hunk 1]
```diff
@@ -186,7 +186,7 @@ containers:
 [1]: /agent/cluster_agent/admission_controller/
 [2]: https://kubernetes.io/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/#capabilities-of-the-downward-api
 [3]: /agent/kubernetes/data_collected/#kube-state-metrics
-[4]: https://github.com/DataDog/integrations-core/blob/master/kubernetes_state/datadog_checks/kubernetes_state/data/conf.yaml.example#L70
+[4]: https://github.com/DataDog/integrations-core/blob/42ecba8180fdce32c2e6739e8db006da2941c29d/kubernetes_state/datadog_checks/kubernetes_state/data/conf.yaml.example#L63
```
Comment (kayayarai) [inline content/en/getting_started/tagging/unified_service_tagging.md:189]:
```suggestion
[4]: https://github.com/DataDog/integrations-core/blob/master/kubernetes_state/datadog_checks/kubernetes_state/data/conf.yaml.example#L63
```

**Review by kayayarai — CHANGES_REQUESTED** [review]
Quite right, but we want the master branch :)

File: content/en/getting_started/tagging/unified_service_tagging.md (hunk 1 above)
Comment (a7i) [reply content/en/getting_started/tagging/unified_service_tagging.md]: The idea was to use a `permalink` so when new code is added, the position doesn't change.

Comment (kayayarai) [reply content/en/getting_started/tagging/unified_service_tagging.md]: Ahhh, gotcha, right. Hm. Let me investigate what we usually do, and I'll fix.

Comment (kayayarai) [reply content/en/getting_started/tagging/unified_service_tagging.md]: So, a glimpse behind the curtain: Our support team gets into tricky situations troubleshooting cases where people have been directed (by the docs for example) to well-intentioned-but-outdated versions of config files via links to specific SHAs. Either way, we have a maintenance problem. So, I've worked around it a bit in this fix https://github.com/DataDog/documentation/pull/15154. 

Thank you so much for pointing out the problem with the link!

Comment (a7i) [reply content/en/getting_started/tagging/unified_service_tagging.md]: that makes sense 👍🏼

Comment (a7i) [reply content/en/getting_started/tagging/unified_service_tagging.md]: better to have the wrong line # than outdated doc

**Pull request merged**